util/embed_builder.py

```diff
@@ -27,9 +27,15 @@ def __init__(
     def build(self) -> discord.Embed:
         embed = discord.Embed(
             title=self.title[:256],
-            description=self.description[:4096],
+            description=self.description[:1024],
```

Comment:
Nononono, this will break loads of stuff. (Especially staffapps). Cut the text off in Wikipedia instead if it's too long. The 4096 cutoff here should only be to prevent errors.